src/components/ModalConfirm/index.js

```diff
@@ -20,7 +20,7 @@ const ModalConfirm = props => (
     </div>
     <div className="modal-body">
       <div className="modal-message">
-        <p>{props.message}</p>
```

Comment:
Removing the `<p>` here appears to mess up the text formatting in the modal message (see the Request in Reading Room confirmation).